village/src/components/SmurfForm.js

```diff
@@ -11,16 +12,20 @@ class SmurfForm extends Component {
   }
 
   addSmurf = event => {
-    event.preventDefault();
-    // add code to create the smurf using the api
-
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
+    event.preventDefault()
+    this.props.postSmurf(this.state)
+    this.props.history.push("/")
+    // this.postSmurf({
+    // 'name':`${this.state.name}`,
+    // 'age':`${this.state.age}`,
+    // 'height': `${this.state.height}`})
```

Comment:
Nice job using the states object instead of shooting it out and interpolating each key value from it